src/configuration.ts

```diff
@@ -129,14 +129,44 @@ export function findConfigurationPath(suppliedConfigFilePath: string, inputFileP
  * @returns a configuration object for TSLint loaded from the file at configFilePath
  */
 export function loadConfigurationFromPath(configFilePath: string): IConfigurationFile {
+    function stripComments(content: string): string {
```

Comment:
Can you move this method to `src/utils.ts`? I'd just rather keep the implementation logic of removing comments separate from the  "business" logic of loading a configuration